### PR TITLE
Feature/atr 371 dev release notes161

### PR DESCRIPTION
**Pull request by k-s-popova**
Please review release notes and PX1013 description

### CODE REVIEW

File: docs/ReleaseNotes.md [hunk 1]
```diff
@@ -1,6 +1,45 @@
...
+| [PX1013](diagnostics/PX1013.md) | The action handler that initiates a background operation or is executed by a background operation must return `IEnumerable`. | Error | Available   | Available   |
+
+### Enhancements and Bug Fixes in the Code Map
+Acuminator 1.6.1 includes the following enhancements in the code map:
+
+ - In the text editor of Visual Studio, the symbol to which the code map navigates is now highlighted.
+ - The tooltip for a data view in the code map now keeps the BQL formatting of the source code.
+ - The namespace prefixes have been removed from the tooltips in the code map.
+ - The categories without child nodes are no longer displayed in the code map.
+ - The code map now displays the actions and methods of the base graph that have been overridden.
```
Comment (SENya1990) [inline docs/ReleaseNotes.md]: @k-s-popova only methods could be overriden, I think actions are redundant here. Also it probably should be specified that "overidden" means override via `PXOverride` attribute. The overrides of virtual methods via standard C# override are not displayed in code map currently

Comment (k-s-popova) [reply docs/ReleaseNotes.md]: fixed

File: docs/ReleaseNotes.md [hunk 2]
```diff
@@ -1,6 +1,45 @@
...
+ - The code map didn't support graphs with complex declarations.
+ - The code map could crash due to an unhandled exception from Roslyn.
+ - The code map performed calculations in hidden mode.
+
+### Other Fixed Bugs
+In this version of Acuminator, the following bugs have been fixed:
+
+ - The PX1018 error was displayed for data views defined in the fluent BQL style.
+ - Fluent BQL joins were not supported by the Acuminator colorizer.
+ - Acuminator didn't recognize graphs if they were used as generic type parameters. <br/>**Fix description**: Graphs as generic type parameters are now supported. Multiple diagnostics (such as PX1000, PX1001, PX1003, and PX1005) have been refactored. The PX1005 diagnostic now supports graph extensions.
```
Comment (SENya1990) [inline docs/ReleaseNotes.md]: The words that PX1005 diagnostic now supports graph extensions imho should be placed in some enhancements section but I'm not sure.

File: docs/ReleaseNotes.md [hunk 3]
```diff
@@ -1,6 +1,45 @@
...
+
+### Other Enhancements
+Acuminator 1.6.1 also includes the following enhancements:
+
+ - Acuminator now supports Visual Studio 2019. 
+ - Acuminator now includes an option to turn off the suppression mechanism for the Acuminator diagnostics. If you set the value of **Tools > Options > Acuminator > Code Analysis > Suppress selected diagnostics** to _False_, Acuminator ignores the suppression file and displays all diagnostics it has found.
```
Comment (SENya1990) [inline docs/ReleaseNotes.md]: @k-s-popova There is another not described option. The Acuminator allows to turn off all static analysis

File: docs/ReleaseNotes.md [hunk 4]
```diff
@@ -1,6 +1,45 @@
...
```
Comment (SENya1990) [inline docs/ReleaseNotes.md]: @k-s-popova the suppression mechanism currently is a feature used only in Acumatica. Maybe we should mention this fact, but I'm not sure. 
@dropsonic, @metallmonster   what do you think?

File: docs/ReleaseNotes.md (hunk 1 above)
Comment (SENya1990) [reply docs/ReleaseNotes.md]: ok

File: docs/ReleaseNotes.md (hunk 2 above)
Comment (k-s-popova) [reply docs/ReleaseNotes.md]: moved to enhancements

File: docs/ReleaseNotes.md (hunk 3 above)
Comment (k-s-popova) [reply docs/ReleaseNotes.md]: added a separate section about two options

File: docs/ReleaseNotes.md (hunk 2 above)
Comment (SENya1990) [reply docs/ReleaseNotes.md]: ok

File: docs/ReleaseNotes.md (hunk 4 above)
Comment (dropsonic) [reply docs/ReleaseNotes.md]: @k-s-popova @SENya1990 let's mention that this is intended for internal use

Comment (SENya1990) [reply docs/ReleaseNotes.md]: I agree

Comment (k-s-popova) [reply docs/ReleaseNotes.md]: done

**Comment by dropsonic** [comment]
@k-s-popova thank you a lot!

**Pull request merged**